tests/detect-mime-email/test.rules

```diff
@@ -0,0 +1 @@
+alert smtp any any -> any any (msg:"Test mime email from"; email.from; content:"toto <toto@gmail.com>"; sid:1;)
```

Comment:
lets add 3 more rules:
`content:"toto"; startswith;` to validate that the buffer starts in exactly the correct position
`content:"com>"; endswith;` validate it ends in the correct position
`bsize:20;` validate buffer size (my calc may be off)